packages/ruleset/src/utils/is-deprecated.js

```diff
@@ -0,0 +1,11 @@
+const isObject = require('./is-object');
+
+/**
+ * Returns `true` if the input is an object with x-sdk-exclude
+ * extension set to true.
```

Comment:
It seems you forgot to reword this comment.

Reply:
done